configure.ac

```diff
@@ -6,7 +6,8 @@ AC_CONFIG_MACRO_DIR([m4])
 dnl m4_include([m4/ax_check_openssl.m4])
 
 AM_INIT_AUTOMAKE
-LT_INIT([disable-static pic-only])
+LT_INIT([disable-static pic-only dlopen])
```

Comment:
What's the reason for `dlopen`?

Reply:
I have added "dlopen" because it is recommended on https://www.gnu.org/software/libtool/manual/html_node/Distributing-libltdl.html :
> You should probably also use the 'dlopen' option to LT_INIT in your configure.ac, otherwise libtool will assume no dlopening mechanism is supported, and revert to dlpreopening, which is probably not what you want.

``LT_INIT([dlopen])`` is used in all examples on that page.

Reply:
Okie.  Not sure what that means on platforms that don't have dlopen, but that will be for another fix, if ever...